pkg/trisa/api/v1beta1/states.go

```diff
@@ -6,6 +6,7 @@ const (
 	TransferStarted          = TransferState_STARTED
 	TransferPending          = TransferState_PENDING
 	TransferRepair           = TransferState_REPAIR
+    TransferReview           = TransferState_REVIEW
```

Comment:
This is a state in Envoy, but wouldn't be used in an exchange; still I included it here just for completeness.

Reply:
This is the "action required" state … but I do plan on changing it's name to review required.